parcel_test.go

```diff
@@ -30,58 +32,132 @@ func getTestParcel() Parcel {
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	// Настройка подключения к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	// Добавляем новую посылку в БД
+	id, err := store.Add(parcel)
+
+	// Проверка на отсутствие ошибки и наличие идентификатора
+	require.NoError(t, err)
+	require.NotEmpty(t, id)
 
 	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	// Получаем только что добавленную посылку, убеждаемся в отсутствии ошибки
+	p, err := store.Get(id)
+	require.NoError(t, err)
+
+	// Проверяем, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	assert.Equal(t, p.Client, parcel.Client)
+	assert.Equal(t, p.Status, parcel.Status)
+	assert.Equal(t, p.Address, parcel.Address)
+	assert.Equal(t, p.CreatedAt, parcel.CreatedAt)
 
 	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	// Удаляем добавленную посылку, убеждаемся в отсутствии ошибки
+	err = store.Delete(id)
+	require.NoError(t, err)
+
+	// Проверяем, что посылку больше нельзя получить из БД
+	_, err = store.Get(id)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для работы с ошибками есть специальные функции
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```